js/renovation/ui/common/utils/get_updated_options.ts

```diff
@@ -94,5 +94,6 @@ export function getUpdatedOptions(
   props: {},
   notDeepCopyArrays: string[] = defaultNotDeepCopyArrays,
 ): ResultItem[] {
-  return objectDiffsWithoutReactProps(oldProps, props, '', notDeepCopyArrays);
+  return objectDiffsWithoutReactProps(oldProps, props, '', notDeepCopyArrays)
+    .filter((item) => !item.path.includes('integrationOptions'));
```

Comment:
May be add 'integrationOptions' into reactProps?

Reply:
reactProps checking is not recursive, so i't not checked in children props.
But idea is great, so I remove integration props during comparing old and new props (not after)